src/pages/PetBio.js

```diff
@@ -27,197 +28,270 @@ function PetBio() {
 
   const findPetObject = petList.find((pet) => pet.id == params.id)
 
-  console.log("petData", findPetObject)
-
   return (
     <>
-      <Box sx={{ marginTop: "4rem" }}>
-        <Grid container spacing={4}>
-          <Grid item xs={5}>
-            <Box sx={{ marginLeft: "6rem", marginTop: "4rem" }}>
-              <Carousel>
-                {findPetObject.photos.map((pet, i) => (
-                  <CarouselPhotos key={i} pet={pet} />
-                ))}
-              </Carousel>
-            </Box>
-          </Grid>
+      <Box
+        sx={{
+          width: {
+            xs: "400px",
+            sm: "600px",
+            md: "800px",
+            lg: "850px",
+            xl: "1000px",
+          },
+          margin: "0 auto",
+          marginTop: "50px",
+        }}
+      >
+        <Typography variant="h3" component="div">
+          My namae is {findPetObject.name}!
```

Comment:
Name is miss spelled

Reply:
I did not see that, thank you so much!! just fixed it